README.md

```diff
@@ -62,79 +49,49 @@ This should bring source and binary files into subdirectories of `/usr/local/go`
 making it possible to access by adding `github.com/tarantool/go-tarantool` in
 the `import {...}` section at the start of any Go program.
 
-<h2>Hello World</h2>
-
-In the "[Connectors](https://www.tarantool.io/en/doc/latest/getting_started/getting_started_go/)"
-chapter of the Tarantool manual, there is an explanation of a very short (18-line)
-program written in Go. Follow the instructions at the start of the "Connectors"
-chapter carefully. Then cut and paste the example into a file named `example.go`,
-and run it. You should see: nothing.
-
-If that is what you see, then you have successfully installed `go-tarantool` and
-successfully executed a program that manipulated the contents of a Tarantool
-database.
-
-<h2>API reference</h2>
+## Documentation
 
-Read the [Tarantool manual](http://tarantool.org/doc.html) to find descriptions
-of terms like "connect", "space", "index", and the requests for creating and
-manipulating database objects or Lua functions.
+Read the [Tarantool documentation](tarantool-doc-data-model-url)
+to find descriptions of terms like "connect", "space", "index", and the
+requests for creating and manipulating database objects or Lua functions.
 
-The source files for the requests library are:
-* [connection.go](https://github.com/tarantool/go-tarantool/blob/master/connection.go)
-  for the `Connect()` function plus functions related to connecting, and
-* [request.go](https://github.com/tarantool/go-tarantool/blob/master/request.go)
-  for data-manipulation functions and Lua invocations.
+In general connector methods can be divided into two main parts:
+- `Connect()` function plus functions related to connecting, and
+- data-manipulation functions and Lua invocations like `Insert()` or `Call()`.
 
-See comments in those files for syntax details:
-```
-Ping
-closeConnection
-Select
-Insert
-Replace
-Delete
-Update
-Upsert
-Call
-Call17
-Eval
-```
+The supported requests have parameters and results equivalent to requests in
+the [Tarantool CRUD operations](tarantool-doc-box-space-url).
+There are also Typed and Async versions of each data-manipulation function.
 
-The supported requests have parameters and results equivalent to requests in the
-Tarantool manual. There are also Typed and Async versions of each data-manipulation
-function.
+### API Reference
 
-The source file for error-handling tools is
-[errors.go](https://github.com/tarantool/go-tarantool/blob/master/errors.go),
-which has structure definitions and constants whose names are equivalent to names
-of errors that the Tarantool server returns.
+Learn API documentation and examples at
+[pkg.go.dev](https://pkg.go.dev/github.com/tarantool/go-tarantool).
 
-## Walking-through example in Go
+### Walking-through example
 
-We can now have a closer look at the `example.go` program and make some observations
+We can now have a closer look at the example and make some observations
 about what it does.
 
```

Comment:
We can now take a closer look at the example and make some observations about what it does.

Reply:
It is not relevant to proposed changes.